fundamentos-do-desenvolvimento-web/bloco-03-int-html-css/dia-01/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+  <head>
+    <meta charset="UTF-8">
+    <title>HTML</title>
+  </head>
+  <body>
+    <h1>AMIZADES DA TRYBE</h1>
```

Comment:
```suggestion
    <h1>👍 AMIZADES DA TRYBE</h1>
```

![](https://media4.giphy.com/media/3o7absbD7PbTFQa0c8/100.webp?cid=5a38a5a2vh043lagxxwgmnq7h10b6qetbv46l3z2wgcr43kz&rid=100.webp&ct=g)